hosts/binarycache/configuration.nix

```diff
@@ -56,7 +57,7 @@
     virtualHosts = {
       "cache.vedenemo.dev" = {
         enableACME = true;
-        forceSSL = true;
+        addSSL = true;
```

Comment:
Why is this change needed again?

Reply:
this was to allow http connections to cache (using ip address), but maybe shouldn't be part of this pr

Reply:
removed this for now